### PR TITLE
Feature/dop refactor

### PR DESCRIPTION
**Pull request by littlejiancc**
#### What type of this PR
/kind feature

#### What this PR does / why we need it:
dop and core-services refactor

### CODE REVIEW

**Comment by sfwn** [comment]
After refactor, unit test Coverage rate from 10.65%  to 10.09%, -0.57%.

File: apistructs/application.go [hunk 1]
```diff
@@ -192,6 +192,8 @@ type ApplicationDTO struct {
 
 	// 应用更新时间
 	UpdatedAt time.Time `json:"updatedAt"`
+
+	Extra string
```
Comment (sfwn) [inline apistructs/application.go:196]: Remove this useless field.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: apistructs/application.go (hunk 1 above)
Comment (littlejiancc) [reply apistructs/application.go]: update to 
``` 
Extra string `json:"-"`
```

**Review by sfwn — APPROVED** [review]

**Pull request merged**